docs/tutorials/hostport.md

```diff
@@ -75,13 +78,16 @@ subjects:
   name: external-dns
   namespace: default
 ---
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: exeternal-dns
 spec:
   strategy:
     type: Recreate
+  selector:
+    matchLabels:
+      app: external-dns
   template:
```

Comment:
This `Deployment` is missing the `template.metadata.labels`. Can we just add them?